core/lib/Thelia/Install/Database.php

```diff
@@ -129,6 +129,15 @@ protected function prepareSql($sql)
     {
         $sql = str_replace(";',", "-CODE-", $sql);
         $sql = trim($sql);
+        preg_match_all('#DELIMITER (.+?)\n(.+?)DELIMITER ;#s', $sql, $m);
+        foreach ($m[0] as $k => $v) {
+            if ($m[1][$k] == '|') {
+                throw new \RuntimeException('You can not use "|" as delimiter: '.$v);
+            }
+            $stored = str_replace(';', '|', $m[2][$k]);
+            $stored = str_replace($m[1][$k], ";\n", $stored);
+            $sql = str_replace($v, $stored, $sql);
+        }
```

Comment:
DELIMITER things not actually needed (DELIMITER is mysql client "reserved word") but I thought that sql should import normal thru `mysql < thelia.sql`
But there was no logic for import such things thru PDO.